hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/Client.java

```diff
@@ -854,7 +854,7 @@ private void handleConnectionFailure(int curRetries, IOException ioe
       if (action.action == RetryAction.RetryDecision.FAIL) {
         if (action.reason != null) {
           LOG.warn("Failed to connect to server: " + server + ": "
-              + action.reason, ioe);
+              + action.reason);
```

Comment:
Do we add the stack as debug if we want to undestand why we failed connecting?

Reply:
Yes! The logs for this part have been moved to the proxy level.